app/helpers/theme_helper.rb

```diff
@@ -36,20 +36,29 @@ def render_theme
     theme.attributes.each do |k, v|
       next unless ATTRIBUTE_MAP.key?(k)
 
-      if k.include?("text") || k.include?("placeholder")
-        hex = get_hex_color_from_theme_value(v)
-        body += "\t--#{ATTRIBUTE_MAP[k]}: #{get_decimal_triplet_from_hex(hex)};\n"
+      if ATTRIBUTE_MAP[k].is_a?(Array)
```

Comment:
Sick Ruby Tricks™ #­54: use [`Kernel#Array`](https://ruby-doc.org/core-3.1.0/Kernel.html#method-i-Array) and you can simply use `.each` regardless whether the thing is an array or not:

```ruby
Array(ATTRIBUTE_MAP[k]).each do |var|
  body += "\t--#{var}: #{get_color_for_key(var, v)};\n"
end
```